tests/shared/test_streamable_http.py

```diff
@@ -776,7 +779,8 @@ async def http_client(basic_server, basic_server_url):
 @pytest.fixture
 async def initialized_client_session(basic_server, basic_server_url):
     """Create initialized StreamableHTTP client session."""
-    async with streamablehttp_client(f"{basic_server_url}/mcp") as (
+    token = "TODO add token here"
```

Comment:
I used code like this to get an oauth token:
```
from databricks import sdk

w = sdk.WorkspaceClient(profile="fieldeng-oauth")
headers = w.config.authenticate()
print(headers)
```

Using a PAT to talk to apps doesn't work